src/inmanta/deploy/state.py

```diff
@@ -198,9 +198,9 @@ class ModelState:
     """
 
     version: int
-    resources: dict["ResourceIdStr", ResourceDetails] = dataclasses.field(default_factory=dict)
+    intent: dict["ResourceIdStr", ResourceIntent] = dataclasses.field(default_factory=dict)
     requires: RequiresProvidesMapping = dataclasses.field(default_factory=RequiresProvidesMapping)
-    resource_state: dict["ResourceIdStr", ResourceState] = dataclasses.field(default_factory=dict)
+    state: dict["ResourceIdStr", ResourceState] = dataclasses.field(default_factory=dict)
```

Comment:
rename 8.

motivation: verbosity + more explicitly differentiate state from intent (important for locking etc).

notes:
- I have to double check but I don't think this requires a db migration script.

personal opinion: small readability improvement => make the change.